src/test/java/net/sf/jabref/exporter/SaveActionsTest.java

```diff
@@ -36,14 +38,14 @@ public void checkSaveActionsParsing() throws IOException {
                 "  ISSN                     = {0886-5930}\n" +
                 "}\n" +
                 "\n" +
-                "@comment{jabref-meta: saveActions:title;IdentityFormatter;}"));
+                "@comment{jabref-meta: saveActions:enabled;title[IdentityFormatter]}"));
 
         ParserResult parserResult = parser.parse();
 
         List<String> saveActions = parserResult.getMetaData().getData(SaveActions.META_KEY);
```

Comment:
I think the first part of this test should be moved to the parser (since it tests the parser more then the save actions). It is more an integration test, that the parser invokes the right methods. 
Here in this test file I would suspect tests like 
`SaveActions saveActions = SaveActions.parse("some test string"); 
Assert.something(expected, saveActions.theRealThing); `